### PR TITLE
Fixes for PHP 7.2 compatibility and typo on call to API function

### PR DESCRIPTION
**Pull request by spficklin**
These are very small fixes to fix PHP v7.2 errors and a typo when calling an API function call.

I realize that upgrading your site to PHP 7.2 just for testing this would be extremely time consuming and will have a lot of side effects.  I personally think just a quick look at the code changes would be a sufficient review.  Otherwise, to test:

To test you need PHP 7.2
1)  Add any job and you get an Drupal error
2)  Try deleting a record using chado_delete_record and be sure that the matching criteria are single values (no arrays)

To test the bad API Call  (PHP 7.2 not required):
3)  Just run the NCBI taxonomy importer. It will fail without this fix because of the bad function call for a function that doesn't exist.

### CODE REVIEW

**Comment by bradfordcondon** [comment]
suggesting @almasaeed2010 as a reviewer since he loves upgrading versions :D

**Comment by bradfordcondon** [comment]
👍 These changes look striaghtforward to me.

I did not test the PHP 7.2 component.

**Review by bradfordcondon — APPROVED** [review]
thanks!  NCBI loader runs and indeed fails on master.

It would be awesome if we could add a simple test for the loader.  Interested?  I can contribute if you'd like...

**Review by laceysanderson — APPROVED** [review]
The code changes look beautiful to me. I didn't test on PHP 7.2 nor have I got the chance to test on my current PHP version. That said, I think such minor changes are good with a single functional review 👍

**Comment by laceysanderson** [comment]
I'm holding off merging though because I Love @bradfordcondon suggestion of a simple test for the loader! Unfortunately I don't have time to contribute one today :-(

**Comment by spficklin** [comment]
A word of warning. Drupal is not yet PHP 7.2 compatible. So, if you upgrade don't do it on a production server, or even an active dev server.  You will get annoying errors constantly.

**Comment by bradfordcondon** [comment]
I've got some time, ill add a test unless @spficklin is actively on it (please let me know).

If I can't contribute it by end of today for some reason let's merge anyway...

**Comment by bradfordcondon** [comment]
i PR'd to stephens branch here rather than push directly to it  https://github.com/tripal/tripal/pull/400

We haven't discussed how review of this sort of thing should go?  I'd think that in this case stephen should soley review the pR to his branch?  but then does this branch need to be re-reviewed?  I dunno....

**Comment by laceysanderson** [comment]
Good point. I think @spficklin should be the only reviewer for your PR #400 and then this PR is re-reviewed if the addition is substantial. In this case, I think we had discussed unit testing being one case where only one review is required and stephen's would count towards that? So maybe we call it good?

**Comment by spficklin** [comment]
I say if the unit test passes then it's good!  Thanks @bradfordcondon for writing that. I will try to remind myself to write these tests for all my pull requests. It's not quite a habit yet for me.

File: tripal_chado/includes/TripalImporter/NewickImporter.inc [hunk 1]
```diff
@@ -108,7 +108,6 @@ class NewickImporter extends TripalImporter {
       '#maxlength' => 255,
     ];
 
-    $type_cv = tripal_get_default_cv('phylotree', 'type_id');
```
Comment (bradfordcondon) [inline tripal_chado/includes/TripalImporter/NewickImporter.inc]: is that why the newick importer broke?  I couldn't figure it out, i swore i had it running at the hackathon.

**Review by almasaeed2010 — APPROVED** [review]

**Comment by almasaeed2010** [comment]
I tested on PHP 7.2 and it worked fine but you are right Drupal keeps crashing or throwing warnings on 7.2.

**Comment by spficklin** [comment]
Thanks @almasaeed2010 !

**Pull request merged**